docs/layer_util.js

```diff
@@ -150,7 +150,7 @@ function toggleLayerOn(layer, map) {
     addLayerFromFeatures(layerDisplayData, index);
     return null;
   }
-  if (layerDisplayData.overlay) {
+  if (layerDisplayData.overlay && !(Array.isArray(layerDisplayData.overlay))) {
```

Comment:
Can we at least make a utility function, either a method on LayerDisplayData or standalone, that says if it's kml, like, isKmlLayer? It would do the array check internally.